gradio/inputs.py

```diff
@@ -108,151 +108,6 @@ def get_template_context(self):
         }
 
 
-class Textbox(InputComponent):
```

Comment:
Instead of deleting the class completely, I would do this so that you can throw a warning appropriately:

```python
class Textbox(InputComponent):
    """
    Component creates a textbox for user to enter input. Provides a string as an argument to the wrapped function.
    Input type: str
    Demos: hello_world, diff_texts
    """

    def __init__(
        self,
        lines: int = 1,
        placeholder: Optional[str] = None,
        default: str = "",
        numeric: Optional[bool] = False,
        type: Optional[str] = "str",
        label: Optional[str] = None,
        optional: bool = False,
    ):
        """
        Parameters:
        lines (int): number of line rows to provide in textarea.
        placeholder (str): placeholder hint to provide behind textarea.
        default (str): default text to provide in textarea.
        numeric (bool): DEPRECATED. Whether the input should be parsed as a number instead of a string.
        type (str): DEPRECATED. Type of value to be returned by component. "str" returns a string, "number" returns a float value. Use Number component in place of number type.
        label (str): component name in interface.
        optional (bool): this parameter is ignored.
        """
        if numeric or type == "number":
            warnings.warn(
                "The 'numeric' type has been deprecated and this parameter has no effect. Use the Number input component instead.",
                DeprecationWarning,
            )
        return components.Textbox(lines=lines, 
                                  placeholder=placeholder, 
                                  default=default, 
                                  label=label,
                                  optional=optional)
```

Reply:
See below for why we shouldn't throw the warning inside the `components.Textbox` class itself